openapi/spec.yaml

```
@@ -16880,6 +16881,58 @@ components:
         status:
           $ref: '#/components/schemas/MachineStatus'
           readOnly: true
+    MachineHealthIssue:
```

YAMLlint (1.38.0)

[error] duplication of key "MachineHealthIssue" in mapping

(key-duplicates)